src/components/FooterWrapper.js

```diff
@@ -76,6 +76,10 @@ const FooterWrapper = ({ children, alwaysShow, fixedHeight }) => {
     a({ href: Nav.getLink('privacy'), style: styles.item }, 'Privacy Policy'),
     div({ style: styles.item }, '|'),
     a({ href: Nav.getLink('terms-of-service'), style: styles.item }, 'Terms of Service'),
+    isRadX() && popoutItem({
```

Comment:
Would it make sense to put this after "Security"? This seems like a more specific part of the general "Security" topic.

Reply:
I agree that swapping the order would make more sense.

Reply:
Good thinking. Changed in [1fc8c16](https://github.com/DataBiosphere/terra-ui/pull/3379/commits/1fc8c16df1831c2997de93e5656e79a97d3394d8).